src/vehicle_use.cpp

```diff
@@ -1767,7 +1767,7 @@ void vehicle::use_washing_machine( int p )
         add_msg( m_bad, _( "You need 5 charges of a detergent for the washing machine." ) );
     } else if( !filthy_items ) {
         add_msg( m_bad,
-                 _( "You need to remove all non-filthy items from the washing machine to start the washing program." ) );
+                 _( "You need to remove all non-filthy items from the washing machine to start the washing program. Hard items such as flashlights can't be washed here." ) );
```

Comment:
This isn't the right place to put this message. This is a warning about hard items, but it won't trigger if you put filthy items in washing machine. You need to create another `else if` case for hard items and put your message there.